test/e2e/conf.js

```diff
@@ -24,14 +24,39 @@ if (process.env.TRAVIS) {
 
   // Mobile devices
   if (process.env.SAUCE_PLATFORM_NAME === 'iOS') {
```

Comment:
Since we're likely to test this on Android in the future, can we pull out the iOS configuration into a separate file? This will help make configuration verbose across platforms.

Also - lets add a comment in our iOS configuration section that links to the available [capabilities](http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities), so contributors can read about the available options.

As an example, `fullReset` might be a useful option to help us reset state between e2e tests.

Reply:
Do you want to have a separate config file for desktop, iOS and android? Or have a common config and split specific configs into separate files which are imported into common config?

Reply:
Ideal state would be to conditionally run our e2e tests for mobile and another for desktop. We _could_ split this into multiple `yarn` commands, but that might be overkill for now.

To make this easy to consume - what about:

```js
if (process.env.SAUCE_PLATFORM_NAME === 'iOS') {
  var appium = require('./appium-config.js');
  config.port = appium.port;
  config.multiCapabilities = appium.iOSCapabilities;
} else {
  // Desktop
  ...
}
```

Where `appium-config.js` is an exported object broken into fields that we care about.